packages/core/src/__integrationtests__/AccountLinking.spec.ts

```diff
@@ -173,10 +170,16 @@ describe('When there is an on-chain DID', () => {
           undefined,
           keytype as KeypairType
         )
-        ;[did, newDid] = await Promise.all([
-          createFullDidFromSeed(paymentAccount, keystore, randomAsHex(32)),
-          createFullDidFromSeed(paymentAccount, keystore, randomAsHex(32)),
-        ])
+        did = await createFullDidFromSeed(
```

Comment:
I guess the next PR could be a refactoring of integration tests to run transactions in batches